.travis.yml

```diff
@@ -2,18 +2,24 @@ sudo: false
 language: node_js
 cache:
   directories:
-    - node_modules
+  - node_modules
 notifications:
   email: false
 node_js:
```

Comment:
since the `.travis.yml` file is changing a lot, maybe this is a good moment to put `node_js: 6` as the line directly after `language: node_js` as I think it's one of the first things you want to know about the configuration.